File: src/server/gkill/api/req_res/update_tag_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/server/gkill/api/message"
	"github.com/mt3hr/gkill/src/server/gkill/dao/reps"
)

type UpdateTagResponse struct {
	Messages message.GkillMessages `json:"messages"`

	Errors message.GkillErrors `json:"errors"`

	UpdatedTag *reps.Tag `json:"updated_tag"`
}

File: src/client/classes/api/req_res/update-tag-response.ts
'use strict'

import { Tag } from '@/classes/datas/tag'
import { GkillAPIResponse } from '../gkill-api-response'

export class UpdateTagResponse extends GkillAPIResponse {

    updated_tag: Tag

    constructor() {
        super()
        this.updated_tag = new Tag()
    }

}


